python/ray/_private/node.py

```diff
@@ -542,6 +547,16 @@ def metrics_export_port(self):
         """Get the port that exposes metrics"""
         return self._metrics_export_port
 
+    @property
+    def runtime_env_agent_port(self):
+        """Get the port that exposes runtime env agent as http"""
+        return self._runtime_env_agent_port
+
+    @property
+    def runtime_env_agent_address(self):
+        """Get the address that exposes runtime env agent as http"""
+        return f"http://{self._raylet_ip_address}:{self._runtime_env_agent_port}"
```

Comment:
We need to use raylet_ip_address because the agent is started by raylet and always co-locates with raylet. However there are some cases I don't have a raylet address so I falled back to node_ip address.

Also, would you mind sharing some cases where node_ip_address != raylet_address?